javascript/ql/src/semmle/javascript/StringOps.qll

```diff
@@ -165,10 +165,10 @@ module StringOps {
 
       StartsWith_Substring() {
         astNode.hasOperands(call.asExpr(), substring.asExpr()) and
-        (call.getMethodName() = "substring" or call.getMethodName() = "substr") and
+        (call.getMethodName() = "substring" or call.getMethodName() = "substr" or call.getMethodName() = "slice") and
```

Comment:
I did a grep: Can you sneakily add the "slice" case to https://github.com/Semmle/ql/blob/f8d0b4e602883b650179b2fd511d451a0a3e0434/javascript/ql/src/semmle/javascript/security/dataflow/ClientSideUrlRedirectCustomizations.qll#L58 while we are at it?

(Or make a note to introduce a substring/substr/slice alias later in a few more locations like I did for path/path.posix/path.win32 in https://github.com/Semmle/ql/pull/2730/commits/5b5f52979d9ea87059e39143f9a425d9131fa63f)